docs/ProductSummaryPrice.md

```diff
@@ -1,4 +1,6 @@
-# Product Summary Price
+# Product Summary Price ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)
+
+:warning: **The Product Summary Price block has been deprecated in favor of the [Product Price](https://vtex.io/docs/components/all/vtex.product-price@1.3.0/) app**. Although support for this block is still granted, we strongly recommend you to update your store theme with the Product Price's blocks in order to keep up with the component's evolution.
```

Comment:
Do you think we can remove the "Although support for this block is still granted" part?